CMakeLists.txt

```diff
@@ -577,6 +585,7 @@ install(TARGETS CHIP
   INCLUDES DESTINATION ${CHIP_CMAKE_INSTALL_INCLUDEDIR}
 )
 
+install(FILES ${CMAKE_BINARY_DIR}/include/CHIPSPVConfig.hh DESTINATION ${INCLUDE_INSTALL_DIR})
```

Comment:
> It's not included everywhere, unfortunately.

It is, isn't it?
```
$ grep Config.hh -r src/
src/Filesystem.hh:#include "Config.hh"
src/common.hh:#include "Config.hh"
```

> I made it installable (renamed to CHIPSPVConfig.hh to avoid file name clashes) and now included from the builtin header.

I don't know why this file needs to be installable. What issue this solves?

Reply:
(This was clarified F2F)